tests/model_inheritance/tests.py

```diff
@@ -157,6 +157,8 @@ def test_create_diamond_mti_common_parents(self):
                 address="1234 W. Ash",
             )
 
+        itrmp.refresh_from_db()
```

Comment:
As far as I'm aware fetching data from DB should not be necessary :thinking:

Reply:
@felixxm actually the same test is passing in my local for postgres. Trying to figure out the actual bug.